src/main/java/uk/gov/hmcts/ecm/common/model/reports/hearingstojudgments/HearingsToJudgmentsSearchResult.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.ecm.common.model.reports.hearingstojudgments;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class HearingsToJudgmentsSearchResult {
+    private Long total;
```

Comment:
Is this class used in ecm-consumer as well? If not, may be adding it in ethos service makes more sense?

Reply:
this is used as part of the report elastic search method in the ccd client so can't be moved to ethos service